Гранкин.md

GrankinAlexey opened the pull request.

GrankinAlexey committed:
Update README.md

GrankinAlexey committed:
Create Alexey Grankin

Исправил

GrankinAlexey committed:
Create Alexey Grankin 2

Decimal

vongostev commented:
Замечаний нет

GrankinAlexey committed:
Гранкин Алексей

Пока что сырая программа, но она работает

vongostev commented:
У меня, к сожалению, нет изображения. В коде не используются преимущества массивов, векторизованные операции, в частности

GrankinAlexey commented:
> У меня, к сожалению, нет изображения. В коде не используются преимущества массивов, векторизованные операции, в частности

1) Отсутствие изображения это на моей стороне проблема или на вашей? У меня всё работает
2) Методы векторизовать не получилось, они как-то странно работали. Постараюсь исправить, этого достаточно для зачëта?

GrankinAlexey commented:
1) Отсутствие изображения это на моей стороне проблема или на вашей? У меня всё работает
2) Методы векторизовать не получилось, они как-то странно работали. Постараюсь исправить, этого достаточно для зачëта?

GrankinAlexey committed:
Гранкин задача N тел 2

Постарался исправить ошибку с изображением. Если не поможет - скину запись экрана

vongostev commented:
У меня так и не получилось запустить. Завтра покажите просто, и все.
